projects/rocr-runtime/runtime/hsa-runtime/image/hsa_amd_mipmap.cpp

```diff
@@ -0,0 +1,277 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Please use this license for new files:

/*
* Copyright © Advanced Micro Devices, Inc., or its affiliates.
*
* SPDX-License-Identifier: MIT
*/